lib/tasks/seek_dev.rake

```diff
@@ -291,9 +291,21 @@ namespace :seek_dev do
     output.close
   end
 
+  task update_thesis_related_publication_types: :environment do
```

Comment:
this needs adding to the upgrade task, and should only print out those that needed updating (i.e haven't already been updated)